generate/config.go

```diff
@@ -117,8 +117,8 @@ func initConfig(filename string) error {
 	}
 	w, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0o644)
 	if err != nil {
-		return err
+		return errorf(nil, "unable to write default genqlient.yaml: %v", err)
```

Comment:
```suggestion
		return errorf(nil, "unable to write default genqlient.yaml: %w", err)
```
Please wrap errors so consumers can use `errors.Is`

Reply:
What would `+` do here?  It seems to me it would only help for ints and `%+q`, but I may not have read the documentation carefully enough.

Reply:
(Do you mean `%#v`?  I don't think we want the quotes here, although below they might be helpful.)

Reply:
If the value is a struct, the `%+v` variant will include the struct’s field names. 
The `%#v` variant prints a Go syntax representation of the value, i.e. the source code snippet that would produce that value.

Reply:
Ah, thanks!  We actually do `err.Error()` before passing it into `Sprintf`, so `%+v` isn't necessary.  (And in general I'd like to aim for more human-readable error messages than even `%+v` would give.)